cmake/OpenCVInstallLayout.cmake

```diff
@@ -0,0 +1,122 @@
+# Following macro updates multiple variables at once:
+#  - OPENCV_BIN_INSTALL_PATH
+#  - OPENCV_LIB_INSTALL_PATH
+#  - OPENCV_CONFIG_INSTALL_PATH
+#  - ...
+macro(ocv_set_install_layout)
+  unset(_var)
+  foreach(arg ${ARGN})
+    if(NOT _var)
+      set(_var ${arg})
+    else()
+      ocv_update(OPENCV_${_var}_INSTALL_PATH ${arg})
+      unset(_var)
+    endif()
+  endforeach()
+endmacro()
+
+if(OPENCV_CUSTOM_INSTALL_LAYOUT)
+
+  include("${OPENCV_CUSTOM_INSTALL_LAYOUT}")
+
+elseif(ANDROID)
```

Comment:
`elseif()` => `endif()`

List OpenCV configuration settings would grow.
So we should have some defaults for them, if custom configuration file doesn't care.

Reply:
I've removed custom layout file. It is easier to use `cmake -C <filename>` for complex configurations.